multiraft/multiraft.go

```diff
@@ -601,16 +601,16 @@ func (s *state) handleRaftReady(readyGroups map[uint64]raft.Ready) {
 				log.Infof("HardState updated: %+v", ready.HardState)
 			}
 			for i, e := range ready.Entries {
-				log.Infof("New Entry[%d]: %s", i, raft.DescribeEntry(e, s.EntryFormatter))
+				log.Infof("New Entry[%d]: %.200s", i, raft.DescribeEntry(e, s.EntryFormatter))
```

Comment:
Yes. Even in these small tests we see very large messages because KeyMax is 4KB; in the real world snapshots in particular will be too large to usefully log.